backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/issue/IssueEntity.java

```diff
@@ -73,6 +74,12 @@
 	@Nullable
 	private BigDecimal budgetedEffort;
 
+	@Nullable
+	private BigDecimal roughEstimation;
```

Comment:
next time pls consider introducing Estimation class instead of using BigDecimal. It's not clear, what's the UoM of estimation, hours, days, points etc?

Reply:
Right, there is an effort UOM on `IssueEntity` but i see your point.